### PR TITLE
libgc 7.6.0 (new formula)

### PR DESCRIPTION
**Pull request by Castux**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/libgc.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+class Libgc < Formula
+  desc "The Boehm-Demers-Weiser conservative garbage collector"
+  homepage "http://www.hboehm.info/gc/"
+  url "http://www.hboehm.info/gc/gc_source/gc-7.2g.tar.gz"
```
Comment (bfontaine) [inline Formula/libgc.rb]: There’s a more recent release at http://www.hboehm.info/gc/gc_source/gc-7.6.0.tar.gz

File: Formula/libgc.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  sha256 "584e29e2f1be4a389ca30f78dcd2c991031e7d1e1eb3d7ce2a0f975218337c2f"
+
+  def install
+    system "./configure", "--disable-silent-rules", "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"test.c").write <<-'EOS'.undent
```
Comment (bfontaine) [inline Formula/libgc.rb]: Use `EOS` instead of `'EOS'`.

**Review by bfontaine — CHANGES_REQUESTED** [review]

**Comment by Castux** [comment]
Thanks for the comments! I had picked 7.2g because the homepage described it as "a fairly recent stable version". I had missed the fact the 7.6.0 is stable as well.

**Review by bfontaine — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks for your first contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!x

**Comment by Castux** [comment]
My pleasure! Homebrew is a real life saver!

**Pull request closed without merging**